test-engine-core-modules/requirements.txt

```diff
@@ -49,6 +49,7 @@ python-dateutil==2.8.2 ; python_version >= "3.10" and python_version < "3.12"
 pytz==2023.3 ; python_version >= "3.10" and python_version < "3.12"
 pyyaml==6.0.1 ; python_version >= "3.10" and python_version < "3.12"
 referencing==0.30.0 ; python_version >= "3.10" and python_version < "3.12"
+rpy2==3.5.13 ; python_version >= "3.10" and python_version < "3.12"
```

Comment:
version here is 3.5.13 while in poetry you specify 3.5.15 minimal

Reply:
ah yeah that is a good one, 3.5.15 doesn't exist actually :s

Reply:
solved in poetry